document/web-api/index.bs

```diff
@@ -118,7 +119,9 @@ Note: This algorithm accepts a {{Response}} object, or a
     1. Let |returnValue| be [=a new promise=]
     1. [=Upon fulfillment=] of |source| with value |unwrappedSource|:
         1. Let |response| be |unwrappedSource|'s [=Response/response=].
-        1. Let |mimeType| be the result of [=extracting a MIME type=] from |response|'s [=response/header list=].
+        1. Let |mimeType| be the result of [=getting=] `` `Content-Type` `` from |response|'s [=response/header list=].
+        1. Remove all [=HTTP tab or space=] from the start and end of |mimeType|.
+        1. [=ASCII lowercase=] |mimeType|.
```

Comment:
You might want to use "if `mimeType` is not a [byte-case-insensitive](https://infra.spec.whatwg.org/#byte-case-insensitive) match".

Reply:
You need to use this suggestion or use byte-lowercase.